From 39a3a6f87c5ef3cb199a0e723df95301ee4c38ff Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Tue, 3 Jul 2012 13:23:56 +0800
Subject: ENGR00180493 SabreSD: register perfmon driver

Fix perfmon:no mxs-perfmon.0 directory after inserting mxs-perfmon.ko.

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 69b5771..2089e2b 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1853,6 +1853,9 @@ static void __init mx6_sabresd_board_init(void)
 	}
 
 	imx6_add_armpmu();
+	imx6q_add_perfmon(0);
+	imx6q_add_perfmon(1);
+	imx6q_add_perfmon(2);
 }
 
 extern void __iomem *twd_base;
-- 
1.7.9.5

